lib/src/rows/Rows.tsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import styled from "styled-components";
+import { RowProps, RowsProps } from "./types";
+
+const DxcRows = ({
+  alignX = "stretch",
+  alignY = "stretch",
+  as = "div",
+  divider = false,
+  gutter = "0rem",
+  reverse = false,
+  children,
+}: RowsProps): JSX.Element => {
+  return (
+    <Rows as={as} alignX={alignX} gutter={gutter} reverse={reverse}>
+      {React.Children.map(children, (child, index) => {
+        return (
+          <>
+            {child.type === DxcRows.Row ? (
+              React.cloneElement(child, { alignY })
+            ) : (
+              <DxcRow alignY={alignY}>{child}</DxcRow>
+            )}
+            {divider && index !== React.Children.count(children) - 1 && <Divider />}
+          </>
+        );
+      })}
+    </Rows>
+  );
+};
+
+const DxcRow = ({ height = "auto", alignY, children }: RowProps): JSX.Element => {
+  return (
+    <Row height={height} alignY={alignY}>
+      {children}
+    </Row>
+  );
+};
+
+DxcRows.Row = DxcRow;
+
+const Rows = styled.div<RowsProps>`
+  display: flex;
+  ${({ alignX, gutter, reverse }) => `
+    flex-direction: ${reverse ? "column-reverse" : "column"};
+    align-items: ${alignX};
+    gap: ${gutter};
+  `}
+  flex-grow: 1;
+`;
+
+const Divider = styled.div`
+  height: 1px;
+  background-color: #999999;
+`;
+
+const Row = styled.div<RowProps>`
+  ${({ height }) =>
+    height === "content"
+      ? "height: fit-content"
+      : height === "auto"
+      ? "flex-grow: 1;"
+      : `height: ${(parseInt(height) / 8) * 100}%`};
+
+  display: flex;
+  align-items: ${({ alignY }) => alignY};
```

Comment:
We should take into account that our values "start" and "end" are not the same as "flex-start" or "flex-end", respectively. This has some connotations that maybe can affect our layout components:

![image](https://user-images.githubusercontent.com/44321109/175947110-69ac39e6-1234-4e84-bdfc-64076793c077.png)

Other is that `flex-start` takes into account the presence of the "-reverse" values of the `flex-direction`, while `start` does not. This only happens at Firefox browser.

See: [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/justify-content), [css-tricks](https://css-tricks.com/snippets/css/a-guide-to-flexbox/#aa-justify-content) and this question at [stack-overflow](https://stackoverflow.com/questions/50919447/flexbox-flex-start-self-start-and-start-whats-the-difference#:~:text=flex%2Dstart%20takes%20into%20account,direction%2C%20while%20start%20does%20not.).

Reply:
What do you think? Should we go for the more generic approach (as it stands right now) or since we are only using flexbox containers (our layout is based on flex) should we use `flex-XXX` properties?

Reply:
I think is better to use the corresponding values for flex.